.npmignore

```diff
@@ -0,0 +1,3 @@
+scripts
```

Comment:
Wow. I did a search in this repo and saw it was unused. That's a very interesting way to do it. I will revert that line.